src/constants.ts

```diff
@@ -0,0 +1,18 @@
+export const PAGINATION_TYPE = {
```

Comment:
Thinking I'm how this would get used:

```js
import {PAGINATION_TYPE} from 'eventbrite';

const {PAGINATION} = PAGINATION_TYPE;
```

That sounds a bit ugly.

How about:

```js
import {PAGINATION_TYPE} from 'eventbrite';
```

? Which would mean we should do:

```js
export const PAGINATION_TYPE = 'page';
export const CONTINUATION_TYPE = 'continuation';
```

Reply:
I am good with that.